eclass/distutils-r1.eclass

```diff
@@ -1994,9 +2006,13 @@ _distutils-r1_post_python_install() {
 		done
 
 		if [[ ${DISTUTILS_USE_PEP517} ]]; then
-			# we need to recompile everything here in order to embed
-			# the correct paths
-			python_optimize "${sitedir}"
+			if ! has_version -b ">=dev-python/gpep517-9"
+			then
+				# TODO: remove when we dep on >=9
```

Comment:
I don't feel that strongly about it but couldn't we just always do this for =no?

Reply:
That was kinda ugly because if you used `python_domodule`, you ended up seeing the byte-compilation message twice.